<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetplayWidget</class>
 <widget class="QDialog" name="NetplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>448</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>701</width>
     <height>461</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>701</width>
     <height>461</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="tbNetplayChat">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="plainText">
     <string>...
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMsg">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>21</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="lwChatWindow">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frDirectIP">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>220</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>321</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbRemotePort">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="leRemoteAddr">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblLocalPort">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Local Port :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRemoteAddr">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Remote Address :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblRemotePort">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Remote Port :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnStartSession">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Start Session</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QSpinBox" name="sbLocalPort">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lblGame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Game : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbSelectedGame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="maxVisibleItems">
     <number>8</number>
    </property>
    <item>
     <property name="text">
      <string>SELECT A GAME</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnStopSession">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Stop Session</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabTraversal">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabJoin">
     <attribute name="title">
      <string>Join</string>
     </attribute>
     <widget class="QPushButton" name="btnTraversalJoin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Join</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>291</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblHostCode">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Host Code :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leHostCode">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHost">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <attribute name="title">
      <string>Host</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QPushButton" name="btnCopyHostCode">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblHostCode2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Host Code :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblHostCodeResult">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>XXXXXXXXX-</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnTraversalHost">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lvlNumSpectators">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Spectators :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbSpectatorCount">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="lblPlayer">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Player :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbLocalPlayer">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <item>
        <property name="text">
         <string>NOT SET</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SPECTATOR</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblInputDelay">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Input Delay :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lePlayerName">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Duck!</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbInputDelay">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbConnMode">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <item>
        <property name="text">
         <string>DIRECT IP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRAVERSAL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblConnMode">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Connection Mode :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblPlayerName">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Nickname : </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>tabTraversal</zorder>
   <zorder>frDirectIP</zorder>
   <zorder>line_2</zorder>
   <zorder>tbNetplayChat</zorder>
   <zorder>btnSendMsg</zorder>
   <zorder>line</zorder>
   <zorder>lwChatWindow</zorder>
   <zorder>lblGame</zorder>
   <zorder>cbSelectedGame</zorder>
   <zorder>btnStopSession</zorder>
   <zorder>gridLayoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
